inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/AbstractSourceOperator.java

```diff
@@ -346,4 +403,222 @@ public void syncSourceFieldInfo(SourceRequest request, String operator) {
     public Integer addDataAddTask(DataAddTaskRequest request, String operator) {
         throw new BusinessException(String.format("not support data add task for type =%s", request.getSourceType()));
     }
+
+    private void updateAgentTaskConfig(SourceRequest request, String operator) {
+        try {
+            if (SourceType.AUTO_PUSH.equals(request.getSourceType())) {
+                return;
+            }
+            final String clusterName = request.getInlongClusterName();
+            final String ip = request.getAgentIp();
+            final String uuid = request.getUuid();
+            if (StringUtils.isBlank(clusterName) || StringUtils.isBlank(ip)) {
```

Comment:
It's better to print a log containing clusterName and ip.

Reply:
Fixed.